src/components/Pages/Location/_Location.scss

```diff
@@ -123,7 +124,7 @@ $coa-location-page-section-background: #f7f7f7;
 
 .coa-LocationPage__table {
   margin: 0;
-  margin-bottom: 1.5rem;
```

Comment:
this fixes the padding issue mentioned in https://austininnovation.slack.com/archives/C8T4YD23T/p1581627704007100